### DISCUSSION TITLE
What motivated you to create this plugin?

### DISCUSSION THREAD

**Discussion by iabdelkareem** [folke/flash.nvim#108] [General]
I'm curious what motivated you to create this plugin especially that it's very similar to leap.nvim also how do you compare it to leap?

**Comment by folke** [comment]
What motivated me to create lazy.nvim when we already have packer.nvim?

Because I can and because I wanted a different way that better aligns with my workflow.

I'm not trying to convince anyone to use flash. leap.nvim is a great plugin as well.

As for a comparison, check the readme of both plugins and try them both yourself.

**Reply by iabdelkareem** [reply]
Fair enough ^^ Thanks
